ui/src/app/pages/admin/admin-institution-edit.tsx

```diff
@@ -168,30 +196,37 @@ export const AdminInstitutionEdit = withUrlParams()(class extends React.Componen
   get fieldsNotEditedAddInstitution() {
```

Comment:
This function actually does the opposite of what it says.  Would you mind changing it to remove the `Not` ?